ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -278,12 +278,12 @@ defaultCodecs :: forall m blk addr.
                 )
               => CodecConfig       blk
               -> BlockNodeToNodeVersion blk
-              -> (addr -> CBOR.Encoding)
-              -> (forall s . CBOR.Decoder s addr)
+              -> (NodeToNodeVersion -> addr -> CBOR.Encoding)
+              -> (NodeToNodeVersion -> forall s . CBOR.Decoder s addr)
               -> NodeToNodeVersion
               -> Codecs blk addr DeserialiseFailure m
                    ByteString ByteString ByteString ByteString ByteString ByteString ByteString
-defaultCodecs ccfg version encAddr decAddr _nodeToNodeVersion = Codecs {
+defaultCodecs ccfg version encAddr decAddr nodeToNodeVersion = Codecs {
```

Comment:
Looks like this is only called in the `ouroboros-consensus` repo's code in the `ouroboros-consensus-diffusion`'s `Ouroboros.Consensus.Node` module and the ThreadNet tests.

This diff alters the ThreadNet call, but it doesn't alter the `Ouroboros.Consensus.Node` call, since that one is using arguments from the Diffusion Layer packages whose type has recently changed, and which is what motivates the change here in the first place.

Reply:
What are you trying to say?